src/ProjectNeon/Assets/Core/Variables/FloatReference.cs

```diff
@@ -21,4 +21,9 @@ public FloatReference(float value)
     {
         return reference.Value;
     }
+
+    public static implicit operator FloatReference(int v)
```

Comment:
What is this change for?

Reply:
@SilasReinagel This was implemented automatically by Visual Studio. I'll remove it